### ISSUE TITLE
Logging errors

### ISSUE THREAD

**Issue by trevormunoz** [trevormunoz/dumbwaiter#7]
When running under Docker on Ubuntu 14.04

```
--- Logging error ---
Traceback (most recent call last):
  File "/usr/local/bin/anaconda/envs/dumbwaiter/lib/python3.4/logging/__init__.py", line 980, in emit
    stream.write(msg)
UnicodeEncodeError: 'ascii' codec can't encode character '\u2026' in position 103: ordinal not in range(128)
Call stack:
  File "/usr/local/bin/anaconda/envs/dumbwaiter/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/bin/anaconda/envs/dumbwaiter/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/dumbwaiter-0.4.0/dumbwaiter/__main__.py", line 29, in <module>
    main()
  File "/dumbwaiter-0.4.0/dumbwaiter/__main__.py", line 26, in main
    dw.load(args.path, host=args.server, port=args.port)
  File "/dumbwaiter-0.4.0/dumbwaiter/pipeline.py", line 434, in load
    actioner = load_and_tranform(source_data)
  File "/dumbwaiter-0.4.0/dumbwaiter/pipeline.py", line 271, in load_and_tranform
    PIPELINE_LOGGER.info('Loading data from source files into memory \u2026')
Message: 'Loading data from source files into memory \u2026'
Arguments: ()
```

**Comment by edsu** [comment]
I wonder if instantiating your RotatingFileHandler with encoding='utf8' will help? Not sure why it works ok on your laptop and not the docker instance. Perhaps some kind of system level locale difference?

**Comment by trevormunoz** [comment]
Yep, looks like that's it. Thanks @edsu !

Fixed by 1f76d94

✈️ Closing issues from the airplane  ✈️️
